release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2069,6 +2069,14 @@ As a result, if you apply a performance profile, all nodes in the cluster reboot
 +
 To revert all nodes in the cluster to the cgroups v2 configuration, you must edit the `Node` resource. For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-clusters-cgroups-2_nodes-cluster-cgroups-2[Configuring Linux cgroup v2]. You cannot revert the cluster to the cgroups v2 configuration by removing the last performance profile. (link:https://issues.redhat.com/browse/OCPBUGS-16976[*OCPBUGS-16976*])
 
+* There is a known issue with using Azure confidential virtual machines, which is a Technology Preview feature in this release. Configuring a cluster to encrypt the managed disk and the Azure VM Guest State (VMGS) blob with a platform-managed key (PMK) or a customer-managed key (CMK) is unsupported. To avoid this issue, only enable encryption of the VMGS blob by setting the value of the `securityEncryptionType` parameter to `VMGuestStateOnly`. (link:https://issues.redhat.com/browse/OCPBUGS-18379[*OCPBUGS-18379*])
```

Comment:
Do we need to tell folks that this parameter is in the `install-config` and give them the full contest: `controlPlane.platform.azure.osdisk,securityProfile.securityEncryptionType`? Or will they be smart enough to look at the bug report?

Reply:
This applies to both install-config and machine sets. I have removed the param from the machine set docs and I believe @bscott-rh removed it from install docs, so this is just in case someone digs around and finds the (currently undocumented) other param that doesn't work.